lib/board.rb

```diff
@@ -0,0 +1,48 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    self.cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(input)
+    cells[input.to_i - 1]
+  end
+
+  def full?
+    cells.all? {|full| full == "X" || full == "O"}
+  end
+
+  def turn_count
+    cells.count{|cell| cell == "X" || cell == "O"}
+  end
+
+  def taken?(input)
```

Comment:
Utilize the implicit return value rather than explicitly returning a value. 

`cells[input.to_i - 1] == "X" || cells[input.to_i - 1] == "O"`